Assets/Scripts/Framework/puzzels/SequencePuzzle/ColorSequenceManager.cs

```diff
@@ -0,0 +1,117 @@
+using UnityEngine;
+using System;
+using UnityEngine.Events;
+using System.Collections.Generic;
+
+public sealed class ColorSequenceManager : MonoBehaviour, SequenceManager<EnumColor>
+{
+    public EnumColor[] Sequence { get; set; }  = new EnumColor[5];
+    public EnumColor[] SubmittedSequence { get; set; } = new EnumColor[5];
```

Comment:
```suggestion
    public EnumColor[] Sequence { get; set; }  = new EnumColor[5];
    public EnumColor[] SubmittedSequence { get; set; } = new EnumColor[5];
```
moeten deze public of kunnen deze ook private (serialized)worden?

Reply:
deze kunnen niet private omdat het komt van de interface